installation/src/Model/SetupModel.php

```diff
@@ -364,7 +364,8 @@ public function validateDbConnection()
 				$options->db_pass_plain,
 				$options->db_name,
 				$options->db_prefix,
-				isset($options->db_select) ? $options->db_select : false
+				isset($options->db_select) ? $options->db_select : false,
+				DatabaseHelper::getEncryptionSettings($options),
```

Comment:
```suggestion
				DatabaseHelper::getEncryptionSettings($options)
```
This comma causes the error.